src/mscorlib/src/Interop/Unix/System.Globalization.Native/Interop.Collation.cs

```diff
@@ -60,6 +60,10 @@ public override bool IsInvalid
 
             protected override bool ReleaseHandle()
             {
+                if (CultureData.InvariantMode)
```

Comment:
This is not really needed now. I had the initial implementation which was keeping the sort handle but now I don't initialize it at all so it is not needed now. I'll convert this to assert instead